docs/features/user-group.md

```diff
@@ -0,0 +1,54 @@
+---
+sidebar_position: 3
+title: "👥 User Groups"
+---
+The User Group feature introduced in Open WebUI allows users with admin access to define and manage access levels for different groups of users. This feature enhances flexibility by providing the ability to assign varying permissions and roles to users based on their needs and responsibilities.
+
+Administrators can manage users and groups directly from the User page from the **Admin Panel**, which includes two main sections: **Users** and **Groups**.
+
+## Managing User Groups
+
+### Roles
+
+There are three default roles for users within a group:
+1. **Pending**: Users awaiting approval for their account to be activated.
+2. **User**: Standard users with permissions defined by the group settings.
+3. **Admin**: Users with access to all features, models, tools, and other system capabilities.
+:::note
+Admins can view the chat history of users, including those with pending access.
```

Comment:
unless specified with the env var to disable this.

Reply:
which env var?

Reply:
look it up.